tests/python_tests/test_sampling.py

```diff
@@ -93,6 +100,22 @@ def test_individual_generation_configs_deterministic(tmp_path, generation_config
     model_id : str = "facebook/opt-125m"
     generate_and_compare_with_hf(model_id, prompts, generation_configs, DEFAULT_SCHEDULER_CONFIG, tmp_path)
 
+@pytest.mark.precommit
+@pytest.mark.skip(reason="Stop strings do not seem to work as expected with beam search in HF, so comparison will fail. If it changes, these cases shall be merged to the test above.")
```

Comment:
You can mark it with xfail instead. It will test that it fails with an expected exception and we also are notified if it starts to work because of `strict=True` arg.